allennlp/service/models/simple_tagger.py

```diff
@@ -0,0 +1,50 @@
+from typing import Dict
+
+from allennlp.data.dataset_readers.sequence_tagging import SequenceTaggingDatasetReader
+from allennlp.data.fields import TextField
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.data.tokenizers import WordTokenizer
+from allennlp.data.vocabulary import Vocabulary
+from allennlp.models.simple_tagger import SimpleTagger
+from allennlp.testing.test_case import AllenNlpTestCase
+from allennlp.service.models.types import Model, JSON
+
+
+# Simple Tagger Model
+def simple_tagger_model() -> Model:
+    """create a simple tagger model."""
+    # this is a bad hack to get the same data as the test case
+    # TODO: replace this
+    test_case = AllenNlpTestCase()
+    test_case.setUp()
+    test_case.write_sequence_tagging_data()
+    dataset = SequenceTaggingDatasetReader(test_case.TRAIN_FILE).read()
+
+    vocab = Vocabulary.from_dataset(dataset)
+    dataset.index_instances(vocab)
+    model = SimpleTagger(embedding_dim=5,
+                         hidden_size=7,
+                         vocabulary=vocab)
+    tokenizer = WordTokenizer()
+
+    def run(blob: JSON):
+        sentence = blob.get("input", "")
```

Comment:
This should be received pre-tokenised in the JSON so it matches the spec for `SimpleTagger.tag`. Also I think the `token_indexers` need to come in via the request and instantiated via `from_params` so that models which use different `token_indexers` can be run as a service.